README.md

```diff
@@ -17,15 +17,26 @@ npm i process-warning
 
 ### Usage
 
-The module exports a builder function that returns a utility for creating warnings and emitting them.
+The module exports two builder functions for creating warnings. A warning has a `emit` method.
 
 ```js
-const warning = require('process-warning')()
+const {
+  createWarning,
+  createDeprecation
+} = require('process-warning')
+
+const warning = createWarning({
+  name: 'FastifyWarning',
+  code: 'FSTWRN001',
+  message: 'Hello %s',
+  unlimited: true
+})
```

Comment:
I think we should avoid referencing Fastify in the examples:

1. Because this is a generic ecosystem module and is merely used by Fastify.
2. Because it can give the reader the impression that this module is only useful in Fastify applications.

```suggestion
const warning = createWarning({
  name: 'ExampleWarning',
  code: 'EXP_WRN_001',
  message: 'Hello %s',
  unlimited: true
})
```